src/agent/multitype_queue/include/db_path.hpp

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+// TODO: should be moved to Config
+constexpr char DEFAULT_FILE_PATH[] = "/home/vagrant/FILE_";
+constexpr char DEFAULT_DB_PATH[] = "queue.db";
```

Comment:
If this is the only define here, move it to `multitype_queue` file.

Reply:
It's also used in the tests.

Reply:
Moved to multitype queue header file.